app/controllers/users_controller.rb

```diff
@@ -0,0 +1,49 @@
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def show
+    @user = User.find(params[:id])
+  end
+
+  def index
+    @users = User.all
+  end
+
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      redirect_to new_user_path
+    else
+      render :new
+    end
+  end
+
+  def edit
+    @user = User.find(params[:id])
+  end
+
+  def update
+    @user = User.find(params[:id])
+
+    if @user.update(article_params)
+      redirect_to @user
```

Comment:
`redirect_to edit_user_path(<USER_ID>)` if user was saved (remember to change <USER_ID> to the code that works)

Reply:
Thank you for the review